fieldtofile/Test/mockServer.ts

```diff
@@ -0,0 +1,47 @@
+import * as mockserver from "mockserver-node";
+import * as mockServerClient from "mockserver-client";
+import * as PH from "processhub-sdk";
+
+export class MockServer {
+  public static async initMockServer(): Promise<void> {
+    await mockserver.stop_mockserver({
+      serverPort: 1080
+    });
+    console.log("Start Mockserver");
+    await mockserver.start_mockserver({
+      serverPort: 1080,
+      trace: true
+    });
+    await MockServer.initResponses();
+    console.log("Started Mockserver");
+  }
+
+  public static async stopMockServer(): Promise<void> {
+    console.log("Shut down Mockserver");
+    await mockserver.stop_mockserver({
+      serverPort: 1080
+    });
+    console.log("Mockserver is offline");
+  }
+
+  private static async initResponses(): Promise<void> {
+    await MockServer.getFiles();
+  }
+
+  private static async getFiles(): Promise<void> {
+
+    await mockServerClient.mockServerClient("localhost", 1080).mockAnyResponse({
```

Comment:
Maybe it would be better to just mock the specific request to detect if the request changes by whatever reason. Currently, even if the download url would be wrong, I guess the test would just continue to pass.

However you prefer it.

Reply:
if the download url is not exactly http://localhost:1080/files/testfile_DEC7AFC8-74FA-43CE-ADAE-CB228FA6BAAE.txt the mockserver will return a 404. So if the download link would change the tests won't pass, because the file would not be created.